treeherder/etl/management/commands/ingest.py

```diff
@@ -153,6 +154,87 @@ def get_decision_task_id(project, revision, root_url):
     return find_task_id(index_path, root_url)
 
 
+def ingestGitPush(options, root_url):
+    project = options["project"]
+    commit = options["commit"]
+    branch = None
+    # Ingesting pushes out of band from the ingestion pipeline would require
+    # a lot of work (not impossible) because the way Servo uses the "auto"
+    # and "try-" branches. A commit can temporarily belong to those branches.
+    # We need to imply the original branch directly from the project name
+    if project.startswith("servo"):
+        branch = project.split("-")[-1]
+        assert branch in ["auto", "try", "master"], \
+            "Valid servo projects are: servo-auto, servo-try, servo-master."
+
+    repository = Repository.objects.filter(name=project)
+    url = repository[0].url
+    splitUrl = url.split('/')
+    owner = splitUrl[3]
+    repo = splitUrl[4]
+    githubApi = "https://api.github.com"
+    baseUrl = "{}/repos/{}/{}".format(githubApi, owner, repo)
+    defaultBranch = fetch_json(baseUrl)["default_branch"]
+    # e.g. https://api.github.com/repos/servo/servo/compare/master...1418c0555ff77e5a3d6cf0c6020ba92ece36be2e
+    compareUrl = "{}/compare/{}...{}"
+    compareResponse = fetch_json(compareUrl.format(baseUrl, defaultBranch, commit))
+    headCommit = None
+    mergeBaseCommit = compareResponse["merge_base_commit"]
+    if mergeBaseCommit:
+        # Since we don't use PushEvents that contain the "before" field [1]
+        # we need to discover the right parent. A merge commit has two parents
+        # [1] https://github.com/taskcluster/taskcluster/blob/3dda0adf85619d18c5dcf255259f3e274d2be346/services/github/src/api.js#L55
+        parents = compareResponse["merge_base_commit"]["parents"]
+        eventBaseSha = None
+        for p in parents:
+            c = fetch_json(p["url"])
```

Comment:
nit: it gives more context to code if you use a word instead of a letter when iterating.

Reply:
I used `c` bc `commit` was already in scope. I used `_commit`. Thanks.